crates/bevy_ui/src/render/mod.rs

```diff
@@ -397,6 +404,7 @@ pub fn extract_uinode_outlines(
     }
 }
 
+#[cfg(feature = "bevy_text")]
```

Comment:
Actually, I think we probably want to modify the logic *inside* this function unfortunately. You still want to be able to render UI even if bevy_text is disabled. Which should clean up the changes above too.

Reply:
I believe I fixed this in b236ab5679287e7f8552fb662d1a2d9753a7651c by simply removing the feature-gate on `bevy_uinodes` and it's associated imports.